plugins/decklink/DecklinkBase.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <map>
+#include <vector>
+#include <mutex>
+
+#include <obs-module.h>
+
+#include "platform.hpp"
+
+#include "decklink-device-discovery.hpp"
+#include "decklink-device-instance.hpp"
+#include "decklink-device-mode.hpp"
+
+class DecklinkBase {
+
+protected:
+	DecklinkBase(DeckLinkDeviceDiscovery *discovery_);
```

Comment:
Style:  I _believe_ the alignment rule is to align on the `*` rather than on the first character after the `*`.  Either this line needs one extra space, or the lines below all need one less space.